network/pulsar-istio.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: k8s.streamnative.io/v1alpha1
+kind: PulsarCoordinator
+metadata:
+  name: private-cloud
+  namespace: pulsar
+spec:
+  image: streamnative/private-cloud:3.1.1.1
+  istio:
+    revision: default
+    trustDomain: cluster.local
+---
+apiVersion: zookeeper.streamnative.io/v1alpha1
+kind: ZooKeeperCluster
+metadata:
+  name: private-cloud
+  namespace: pulsar
+  labels:
+    k8s.streamnative.io/coordinator-name: private-cloud
+spec:
+  image: streamnative/private-cloud:3.1.1.1
+  replicas: 3
+  pod:
+    labels:
+      sidecar.istio.io/inject: "true"
```

Comment:
The label is unnecessary as the operator will handle the Istio injection lables.

Reply:
@ciiiii removed.